types/react-tag-input/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React-Tags (react-tag-input) 5.0
+// Type definitions for React-Tags (react-tag-input) 6.1.0
```

Comment:
this should just read 6.1

Reply:
Thanks for the quick review! Just made that change so Travis should re-run.